.eslintrc.json

```diff
@@ -1,11 +1,25 @@
 {
     "plugins": ["node"],
-    "extends": ["eslint:recommended", "plugin:node/recommended"],
+    "extends": ["airbnb-base", "plugin:node/recommended"],
     "rules": {
+        "consistent-return": 0,
```

Comment:
Is this list driven by preferences, or is this what was currently failing in the existing code?

Reply:
currently failing. I'm going to remove them one at a time for easy review.